### PR TITLE
added env variables

### PR DESCRIPTION
**Pull request by elizabethhalper**
testing with test env var and will add each variable in slowly

### CODE REVIEW

File: LoginService/LoginService/Startup.cs [hunk 1]
```diff
@@ -38,7 +38,9 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            string connectionString = Configuration.GetConnectionString("DefaultConnection");
+            string connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
+            string testing = Environment.GetEnvironmentVariable("TEST_VAL_KEY");
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply LoginService/LoginService/Startup.cs]: done

File: LoginService/LoginService/Startup.cs [hunk 2]
```diff
@@ -56,17 +58,16 @@ public void ConfigureServices(IServiceCollection services)
...
                 {
-                    Configuration["JwtKey"] = GetSecret("token-sign-key");
+                    Environment.GetEnvironmentVariable("JWT_KEY");
                 }
                 catch(Exception x) // until secrets work end-to-end have plan B
                 {
                     _telemetryClient.TrackException(x);
-
-                    Configuration["JwtKey"] = Guid.NewGuid().ToString();
+                    Environment.GetEnvironmentVariable("JWT_KEY");
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply LoginService/LoginService/Startup.cs]: changed it back to original code. this is run if we are unable to obtain the Jwt Key through environment vars

File: LoginService/chart/values.yaml [hunk 3]
```diff
@@ -25,6 +25,28 @@ hexadite:
   keyVault: https://microservices-deploy-kv.vault.azure.net/
 
 service:
-  type: ClusterIP
+  type: LoadBalancer
```
Comment (wesyao) [inline LoginService/chart/values.yaml]: LoadBalancer was good for testing, but we're going to need this to be ClusterIP

File: LoginService/chart/values.yaml [hunk 4]
```diff
@@ -5,9 +5,9 @@
 app:
   replicaCount: 3
   image:
-    repository: acrmicroserviceswestus.azurecr.io # ACR Login Server
-    imageName: containerized-microservices-pipeline-service
-    tag: canary
+    repository: acrmicrok8eastus.azurecr.io # ACR Login Server
+    imageName: lizmttest
```
Comment (wesyao) [inline LoginService/chart/values.yaml]: Revert back to original image

File: LoginService/chart/values.yaml [hunk 5]
```diff
@@ -5,9 +5,9 @@
 app:
   replicaCount: 3
   image:
-    repository: acrmicroserviceswestus.azurecr.io # ACR Login Server
-    imageName: containerized-microservices-pipeline-service
-    tag: canary
+    repository: acrmicrok8eastus.azurecr.io # ACR Login Server
+    imageName: lizmttest 
+    tag: latest
```
Comment (wesyao) [inline LoginService/chart/values.yaml]: Revert back to original tag

File: LoginService/LoginService/appsettings.json [hunk 6]
```diff
@@ -7,7 +7,7 @@
   "SecretsVaultUrl": "https://ivanbny-mt-svc-kv.vault.azure.net",
```
Comment (wesyao) [inline LoginService/LoginService/appsettings.json]: @ivan-shaporov This KV is old right? Do we need to actually check this in?

File: LoginService/LoginService/Startup.cs [hunk 7]
```diff
@@ -149,7 +149,7 @@ private string GetSecret(string secretName)
         {
             using (var kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(GetTokenAsync)))
             {
-                var sec = kv.GetSecretAsync($"{Configuration["SecretsVaultUrl"]}/secrets/{secretName}").Result;
+                var sec = kv.GetSecretAsync($"{Environment.GetEnvironmentVariable("SECRETS_VAULT_URL")}/secrets/{secretName}").Result;
```
Comment (wesyao) [inline LoginService/LoginService/Startup.cs]: Will these lines of code crash if the environment variable isn't there?

File: LoginService/LoginService/Controllers/LoginController.cs [hunk 8]
```diff
@@ -96,7 +95,7 @@ public string Get(string value)
 
             var key = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(_configuration["JwtKey"]));
             var creds = new SigningCredentials(key, SecurityAlgorithms.HmacSha256);
-            var expires = DateTime.UtcNow.AddMinutes(Convert.ToDouble(_configuration["JwtExpireMinutes"]));
+            var expires = DateTime.Now.AddMinutes(Convert.ToDouble(_configuration["JwtExpireMinutes"]));
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply LoginService/LoginService/Controllers/LoginController.cs]: It had thrown an error with using GetEnvironmentVariable, but I changed it back

File: LoginService/LoginService/Startup.cs [hunk 9]
```diff
@@ -78,7 +77,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.AddPolicy("AllowSpecificOrigin", builder =>                
                     builder
-                        .WithOrigins(Configuration["CorsOrigins"])
+                                  .WithOrigins(Configuration["CorsOrigins"])
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply LoginService/LoginService/Startup.cs]: done

File: LoginService/LoginService/Startup.cs [hunk 10]
```diff
@@ -121,6 +119,7 @@ public void ConfigureServices(IServiceCollection services)
                 });
 
             services.AddApplicationInsightsTelemetry(Configuration);
+            //TODO: Figure out how to use ConfigMaps with App Insights Telemetry
```
Comment (billpratt) [inline LoginService/LoginService/Startup.cs]: Do we have a ticket in the backlog to address this TODO?

**Comment by elizabethhalper** [comment]
@billpratt yep, I had a test variable that worked, I just took it out for the PR

**Pull request merged**